bot_cli/commands.py

```diff
@@ -248,9 +270,76 @@ def birthdays(args):
 
     if get_birthdays_per_week:
         result = "Next week birthdays:\n" + "-" * 10 + "\n"
-        result += "\n".join([f"{day}: {celebrate_users}" for day, celebrate_users in get_birthdays_per_week.items()])
+        result += "\n".join(
+            [
+                f"{day}: {celebrate_users}"
+                for day, celebrate_users in get_birthdays_per_week.items()
+            ]
+        )
         result += "\n" + "-" * 10
         print(Fore.YELLOW + result)
     else:
-        print(Fore.YELLOW +
-              "There is no one to celebrate birthday next week")
+        print(Fore.YELLOW + "There is no one to celebrate birthday next week")
+
+
+@note_error_handler
+def add_note(notebook: Notes, args):
+    text = " ".join(args)
+    if text.isspace() or len(text) < MIN_NOTE_LEN:
+        raise ValueError(
+            f"Note cannot be empty and must be more than {MIN_NOTE_LEN} characters long"
+        )
+    note_id, _ = notebook.add_note(text)
+    print_success(f"Note with id {note_id} created successfully")
+
+
+# command, index, text = args[0], args[1], " ".join(args[2:])
+@note_error_handler
+def update_note(notebook: Notes, args):
+    index, text = args[0], " ".join(args[1:])
+    notebook.update_note(int(index), text)
+    print_success("Note successfully updated")
+
+
+@note_error_handler
+def replace_note(notebook: Notes, args):
```

Comment:
Would like a name to be `change_note` to correspond to the command `change-note`

Reply:
change everywhere replace_note -> change_note (cli, cls method,  and commadn)